frontend_lib/i18next.scanner/fr/translation.json

```
@@ -33,6 +34,12 @@
   "Search user...": "Rechercher un utilisateur...",
   "Unknown error, please inform an administrator": "Erreur inconnue, veuillez contacter un administrateur",
   "Error detail": "Détail de l'erreur",
+  "Error while archiving content": "Erreur lors de l'archivage d'un contenu",
+  "Error while changing status": "Erreur lors d'un changement de statut",
+  "Error while deleting content": "Erreur lors de la suppression d'un contenu",
+  "Error while restoring content": "Erreur lors de la restauration du contenu",
+  "Error while saving new comment": "Erreur lors de l'enregistrement d'un nouveau commentaire",
+  "Error while saving new title": "Erreur lors de l'enregistrement d'un nouveau titre",
```

```suggestion
  "Error while saving new title": "Erreur lors de l'enregistrement du nouveau titre",
```

"Error while changing the title?"